src/main/java/app/RunGestureBridgeApp.java

```diff
@@ -1,10 +1,16 @@
 package app;
 
 import data_access.SignLanguageTranslationDataAccessObject;
+import data_access.VoiceDataAccessObject;
+import entity.AudioSettings;
+import entity.AudioSettingsFactory;
```

Comment:
I'm not sure if the builder is allowed to call the entity directly, there maybe a dependency issue here?